mern/client/src/webpack.config.js

```diff
@@ -0,0 +1,5 @@
+const Dotenv = require("dotenv-webpack");
```

Comment:
Remember that this env file is used to contain the server route to MongoDB. Currently set to localhost, but needs to be changed to http address from Google Cloud Run.